### PR TITLE
PARQUET-1033: Improve documentation about WriteBatchSpaced

### PR DESCRIPTION
**Pull request by xhochy**

### CODE REVIEW

File: src/parquet/column/writer.h [hunk 1]
```diff
@@ -166,8 +166,33 @@ class PARQUET_EXPORT TypedColumnWriter : public ColumnWriter {
...
+  /// @param def_levels The Parquet definiton levels, length is num_values
+  /// @param rep_levels The Parquet repetition levels, length is num_values
+  /// @param valid_bits Bitmap that indicates if the row is null on the lowest nesting
+  ///   level. The length is number of rows in the lowest nesting level.
+  /// @param valid_bits_offset The offset in bits of the valid_bits where the
+  ///   first relevant bit resides.
+  /// @param values The values in the lowest nested level including
+  ///   spacing for nulls on the lowest levels; input has the length
+  ///   of the number of rows on the lowest nesting level.
+  /// @param values The
```
Comment (majetideepak) [inline src/parquet/column/writer.h]: This last line should be removed?

Comment (xhochy) [inline src/parquet/column/writer.h]: Done.

Comment (majetideepak) [inline src/parquet/column/writer.h]: Yet to push the commit?

**Comment by majetideepak** [comment]
+1 LGTM after pushing the commit.

**Pull request closed without merging**